package.json

```diff
@@ -4,7 +4,7 @@
   "license": "Apache-2.0",
   "private": true,
   "dependencies": {
-    "@arbitrum/sdk": "^2.0.15",
+    "@arbitrum/sdk": "^v3.0.0",
```

Comment:
if you run `yarn add @arbitrum/sdk@latest` it should be `"@arbitrum/sdk": "^3.0.0"`

Reply:
Fixed! 
Thanks Fionna!